src/Cake.Common/Tools/InspectCode/InspectCodeAliases.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.InspectCode
+{
+    /// <summary>
+    /// Contains functionality related to Resharper's code inspection.
+    /// </summary>
+    [CakeAliasCategory("InspectCode")]
```

Comment:
Yes, These methods would be grouped like ReSharper/InspectCode which would be very nice.